SU-SWS/stanford_jumpstart_academic
# HSD8-925: Added 4 JSA feature exporter modules

# READY FOR REVIEW

# Summary
Created 4 feature modules with views exporters for specific sets of data ([see ticket](https://stanfordits.atlassian.net/browse/HSD8-925) for views links). **NOTE:** The `stanford_jsa_d8_exporter_events` feature module contained in this PR was [already a feature on the site](https://creees.stanford.edu/admin/structure/features/stanford_events_export). However, it had been overridden. I am not sure where this `stanford_events_export` feature originates from.

# Needed By (Date)
11/3

# Steps to Test
1. Pull code locally and rebuild cache
2. Open the extend/modules page and make sure all 4 new modules are unavailabe/hidden there.
3. Enable the 4 new modules through drush `drush en` -- modules are:
    -  `stanford_jsa_d8_exporter_events`
    -  `stanford_jsa_d8_exporter_news`
    -  `stanford_jsa_d8_exporter_publications`
    -  `stanford_jsa_d8_exporter_people`
4. Make sure the associated views are properly created and the paths work (I had to clear the cache after enabling module to get the page to show up). Paths are:
    -  `stanford_jsa_d8_exporter_events - `/events-xml`
    -  `stanford_jsa_d8_exporter_news - `/news-xml`
    -  `stanford_jsa_d8_exporter_publications - `/publications-xml`
    -  `stanford_jsa_d8_exporter_people - `/people-xml`

# Affected Projects or Products
H&S

# Associated Issues and/or People
https://stanfordits.atlassian.net/browse/HSD8-925

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
